.github/workflows/documentation.yml

```
@@ -31,6 +31,9 @@ jobs:
       - name: Push to Github Pages branch
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DOCSEARCH_API_KEY: ${{ secrets.DOCSEARCH_API_KEY }}
+          DOCSEARCH_APP_KEY: ${{ secrets.DOCSEARCH_APP_ID }}
```

oops i made a mistake lol

```suggestion
          DOCSEARCH_API_KEY: ${{ secrets.DOCSEARCH_API_KEY }}
          DOCSEARCH_APP_ID: ${{ secrets.DOCSEARCH_APP_ID }}
```